From 93def973cbed509d6f7f61f373ee075d16c321be Mon Sep 17 00:00:00 2001
From: Jamie Mansfield <malgm@lapis.blue>
Date: Sun, 26 Apr 2015 22:00:52 +0100
Subject: [PATCH] Change author of the theme


diff --git a/src/main/java/com/atlauncher/gui/theme/Theme.java b/src/main/java/com/atlauncher/gui/theme/Theme.java
index 0547030..0250c0b 100644
--- a/src/main/java/com/atlauncher/gui/theme/Theme.java
+++ b/src/main/java/com/atlauncher/gui/theme/Theme.java
@@ -31,9 +31,9 @@ public final class Theme {
     /**
      * This is the default theme used by the launcher incase all else fails and it cannot read the theme from disk.
      */
-    public static final Theme DEFAULT_THEME = new Theme(Constants.LAUNCHER_NAME, "RyanTheAllmighty", true, new Color(40, 45, 50)
-            , new Color(255, 255, 255), new Color(0, 0, 0), new Color(0, 136, 204), new Color(100, 100, 200), new
-            Color(80, 170, 107), new Color(50, 55, 60), new Color(50, 55, 60), new Color(30, 35, 40), new Color(255,
+    public static final Theme DEFAULT_THEME = new Theme(Constants.LAUNCHER_NAME, "jamierocks", true, new Color(40,
+            45, 50) , new Color(255, 255, 255), new Color(0, 0, 0), new Color(0, 136, 204), new Color(100, 100, 200),
+            new Color(80, 170, 107), new Color(50, 55, 60), new Color(50, 55, 60), new Color(30, 35, 40), new Color(255,
             255, 255), new Color(255, 0, 0), new Color(255, 255, 255), new Color(137, 194, 54), new Color(255, 255,
             76), new Color(238, 34, 34), new Color(255, 0, 255), "SansSerif", "SansSerif", "Oswald-Regular",
             "SansSerif");
-- 
1.9.5 (Apple Git-50.3)

